src/Claim.ts

```diff
@@ -1,17 +1,21 @@
-import * as crypto from 'crypto'
+/* tslint:disable:no-relative-imports */
```

Comment:
What happened here? Didn't changing `import { IllegalArgumentException } from './Exceptions'` to `import { IllegalArgumentException } from 'Exceptions'` work?

Reply:
@wzalazar I don't understand whether you're saying that my proposed change works or not :stuck_out_tongue:

Reply:
in the root of the project, should work with absolute paths :)

Reply:
But, I understand this is a library. For this reason when you install this package in your host project maybe it will fail.